spec/OBD/OBD.vspec

```diff
@@ -68,85 +75,99 @@
   description: PID 04 - Engine load in percent - 0 = no load, 100 = full load
 
 - CoolantTemperature:
-  datatype: float
+  datatype: int16
   type: sensor
   unit: celsius
+  min: -40
+  max: 215
   description: PID 05 - Coolant temperature
 
 - ShortTermFuelTrim1:
-  datatype: int8
+  datatype: float
   type: sensor
   unit: percent
   min: -100
-  max: 100
+  max: 99.2
```

Comment:
And due to the OBD-representation the max value is actually +99.2 (sent as 0xFF, meaning (100*255/128)-100 = 99.2)